core/tests/test_views.py

```diff
@@ -788,3 +788,86 @@ def test_moderator_dashboard_filter_no_matching_authors(
     assert response.status_code == 200
     reported = response.context["reported_posts"]
     assert len(reported) == 0
+
+
+def generate_test_image(
+    file_format: str = "JPEG", size: tuple = (300, 300), color: tuple = (255, 0, 0)
+) -> SimpleUploadedFile:
+    image = Image.new("RGB", size, color)
+    byte_io = io.BytesIO()
+    image.save(byte_io, file_format)
+    byte_io.seek(0)
+    return SimpleUploadedFile("test_image.jpg", byte_io.read(), content_type="image/jpeg")
+
+
+@pytest.mark.django_db()
+def test_avatar_validation_rejects_invalid_dimensions() -> None:
+    from core.forms import CommunityForm
+
+    invalid_avatar = generate_test_image(size=(100, 100))
+    form = CommunityForm(data={"name": "test", "privacy": "10_PUBLIC"}, files={"avatar": invalid_avatar})
+    assert not form.is_valid()
+    assert "avatar" in form.errors
+
+
+@pytest.mark.django_db()
+def test_save_method_processes_images_correctly(user: User) -> None:
+    from core.forms import CommunityForm
+
+    avatar = generate_test_image(size=(300, 300))
+    background = generate_test_image(size=(1920, 384))
+    form = CommunityForm(
+        data={"name": "Test Community", "privacy": "10_PUBLIC"},
+        files={"avatar": avatar, "background": background},
+    )
+    assert form.is_valid()
+    community = form.save(commit=False)
+    community.author = user
+    community.save()
+    assert community.avatar
```

Comment:
ok, we know that both files have been saved. 
Can you proof, in an easy way, that those are images and have proper size?